man/resolved.conf.xml

```diff
@@ -138,21 +138,25 @@
 
       <varlistentry>
         <term><varname>DNSSEC=</varname></term>
-        <listitem><para>Takes a boolean argument or
-        <literal>allow-downgrade</literal>. If true all DNS lookups are
-        DNSSEC-validated locally (excluding LLMNR and Multicast
-        DNS). If the response to a lookup request is detected to be invalid
-        a lookup failure is returned to applications. Note that
-        this mode requires a DNS server that supports DNSSEC. If the
-        DNS server does not properly support DNSSEC all validations
-        will fail. If set to <literal>allow-downgrade</literal> DNSSEC
-        validation is attempted, but if the server does not support
-        DNSSEC properly, DNSSEC mode is automatically disabled. Note
-        that this mode makes DNSSEC validation vulnerable to
-        "downgrade" attacks, where an attacker might be able to
-        trigger a downgrade to non-DNSSEC mode by synthesizing a DNS
-        response that suggests DNSSEC was not supported. If set to
-        false, DNS lookups are not DNSSEC validated.</para>
+        <listitem><para>Be warned that DNSSEC is an experimental 
+        feature, and has known vulnerabilities, so use at your own risk.
```

Comment:
It's not DNSSEC that is experimental and has vulnerabilities, but this specific implementation. Suggestion:

```suggestion
        <listitem><para>Be warned that DNSSEC support is experimental
        and has known validation vulnerabilities, so use at your own risk.
```

Reply:
Yeah, that sounds more accurate